src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -49,7 +49,34 @@ <h4 mat-dialog-title>Register Workflow</h4>
     </mat-step>
     <mat-step>
       <ng-template matStepLabel>Create a workflow</ng-template>
-      <div *ngIf="selectedOption?.value == 0">
+      <div *ngIf="selectedOption?.value === 0">
+        <p>
+          Navigate to GitHub to install our GitHub app on your repositories/organizations. See our
+          <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps.html" target="_blank" rel="noopener noreferrer">
```

Comment:
I'm sure your doc covers it, but I wonder if it's worth stating here that a .dockstore.yml will be required? People may have an existing workflow in pipeline, come into Dockstore, pick the recommended way to add their workflow, and then... nothing -- because they don't have .dockstore.yml.

We do want to force people to go this way, and your doc will cover it, so maybe I'm overthinking it.

Reply:
I'll add a note that a .dockstore.yml is required.